products/extensions/b2b-components/order-approval/03-payment-process.md

```diff
@@ -0,0 +1,43 @@
+---
+nav:
+  title: Payment Process
+  position: 30
+
+---
+
+# Order approval's Payment Process
+
+## Introduction
+
+The payment process of the order approval component is the same as the payment process of the order component, you can select the payment method that you want to use for your orders, with the exception that if this is the online payment method (Visa, PayPal, etc.), the payment process will be executed only after the order has been approved.
```

Comment:
```suggestion
# Order approval's payment process

The payment process of the order approval component is the same as the payment process of the order component. You can select the payment method that you want to use for your orders; however unlike the standard order component, the payment process will be executed only after the order has been approved in case of the online payment method (Visa, PayPal, etc.).
```